Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.5-alpine
+FROM ruby:3.1
```

Comment:
@abhinav-nain , Is there a reason for not using 3.1-alpine ?

Reply:
Yes, it seems like the 3.1-alpine image has some dev tools issue and the gem installation fails